interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+<?php   
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="claims";
+    require_once "../../../../globals.php";
+   
+    require_once '../src/ClaimSearchModel.php';
+    require_once '../src/ClaimSearch.php';
+    require_once '../src/ClaimRevApi.php';
+    require_once '../src/UploadEdiFileContentModel.php';
+
+   
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearch;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearchModel;
+
+    class ClaimsPage
```

Comment:
classes should be in src directory with other classes

Reply:
done